include/oneapi/dpl/internal/random_impl/uniform_real_distribution.h

```diff
@@ -226,11 +228,10 @@ class uniform_real_distribution
     generate(_Engine& __engine, const param_type& __params)
     {
         auto __engine_output = __engine();
-        auto __res = static_cast<scalar_type>(__engine_output);
-        __res = ((__res - static_cast<scalar_type>(__engine.min())) /
-                 (1 + static_cast<scalar_type>(__engine.max() - __engine.min()))) *
-                    (__params.b() - __params.a()) +
-                __params.a();
+        scalar_type __res =
```

Comment:
if we change 1 line on a loop, will we call and compute static_cast<scalar_type>(1 + (__engine.max() - __engine.min()))) each time?